src/main/kotlin/org/wycliffeassociates/otter/jvm/app/ui/resourcetakes/viewmodel/TabRecordableViewModel.kt

```diff
@@ -0,0 +1,23 @@
+package org.wycliffeassociates.otter.jvm.app.ui.resourcetakes.viewmodel
+
+import javafx.beans.binding.Bindings
+import javafx.beans.binding.StringBinding
+import javafx.beans.property.SimpleStringProperty
+import org.wycliffeassociates.otter.common.data.workbook.Take
+import org.wycliffeassociates.otter.jvm.app.ui.takemanagement.viewmodel.AudioPluginViewModel
+import org.wycliffeassociates.otter.jvm.app.ui.takemanagement.viewmodel.RecordableViewModel
+import java.util.concurrent.Callable
+
+class TabRecordableViewModel(
+    val labelProperty: SimpleStringProperty,
+    audioPluginViewModel: AudioPluginViewModel
+): RecordableViewModel(
+    audioPluginViewModel
+) {
+    fun getFormattedTextBinding(): StringBinding = Bindings.createStringBinding(
+        Callable { getFormattedText() },
+        recordableProperty
+    )
+
+    private fun getFormattedText(): String? = recordable?.textItem?.text
```

Comment:
what is recordable? it doesn't look like it's declared?
I see this wasn't a simple rename of TabRecordableViewModel from RecordableTabViewModel, and a lot was removed

Reply:
TabRecordableViewModel inherits from RecordableViewModel now, and RecordableViewModel has `recordable`. That's also why a lot was removed- the functionality is in the base class RecordableViewModel